server/tests/modules/test_reorder.py

```diff
@@ -0,0 +1,95 @@
+from django.test import TestCase
+from server.tests.utils import *
+from server.execute import execute_nocache
+import json
+import pandas as pd
+import numpy as np
+import io
+
+
+class ReorderFromTableTests(LoggedInTestCase):
+    # Current dtype choices: "String|Number|Date"
+    # Current direction choices: "Select|Ascending|Descending"
+    # If the position of the values change, tests will need to be updated
+
+    # NaN and NaT always appear last as the policy in SortFromTable dictates
+
+    def setUp(self):
+        super(ReorderFromTableTests, self).setUp()
+        self.test_csv = (
+            'name,date,count,float\n'
+            + 'Dolores,2018-04-22,3,13.5\n'
+            + 'Bernard,2018-04-23,not_a_count,2.0\n'
+            + 'Ford,2016-10-02,5,\n'
+            + 'Dolores,0_not_a_date,4,2.8'
+        )
+        # A reference table for correctness checking
+        self.table = pd.read_csv(io.StringIO(self.test_csv), dtype={
+            'name': object,
```

Comment:
Types will probably come through correctly in this test without explicitly setting them here. Try it!